releasenotes/notes/deprecate-classical-function-3bf44ef26d984366.yaml

```diff
@@ -1,5 +1,9 @@
 ---
 deprecations_circuits:
   - |
-    The :class:`.ClassicalFunction` and its related :meth:`.classical_function`
+    The :mod:`qiskit.circuit.classicalfunction` and with it
+    the :class:`.ClassicalFunction` and its related :func:`.classical_function`
     have been deprecated as of Qiskit 1.4 and will be removed in Qiskit 2.0.
+    This change is done to stop relying on the `tweedledum` external library.
```

Comment:
```suggestion
    This change is done to avoid a dependency on the external library `tweedledum`,
    which is no longer compatible with all of Qiskit's supported platforms and 
    Python versions.
```